concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/contractexample/cis2nft/Cis2NftParameters.java

```diff
@@ -0,0 +1,140 @@
+package com.concordium.sdk.examples.contractexample.cis2nft;
+
+import com.concordium.sdk.examples.contractexample.parameters.*;
+import com.concordium.sdk.transactions.ReceiveName;
+import com.concordium.sdk.transactions.smartcontracts.Schema;
+import com.concordium.sdk.transactions.smartcontracts.SchemaParameter;
+import com.concordium.sdk.transactions.smartcontracts.SchemaVersion;
+import com.concordium.sdk.types.*;
+import lombok.SneakyThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Helper class for creating and initializing parameters being used in {@link Cis2Nft}.
+ */
+public class Cis2NftParameters {
```

Comment:
Maybe add some documentation that links to the smart contract source code. This will help a user "bridge" the rust parameters with the java parameters I think.